assets/scss/404.scss

```diff
@@ -0,0 +1,108 @@
+@import url("https://fonts.googleapis.com/css2?family=Nunito:wght@300;400&display=swap");
+
+@mixin transbox($trans){
+    transform-box: $trans
+}
+
+@keyframes swing {
+	0% {
+		transform: rotate(10deg);
+	}
+	100% {
+		transform: rotate(-10deg);
+	}
+}
+@keyframes swinghair {
+	0% {
+		transform: rotate(6deg);
+	}
+	100% {
+		transform: rotate(-6deg);
+	}
+}
+/* go back btn */
+/* responsiveness */
+body {
+	background: linear-gradient(to right, #02aab0, #00cdac);
+}
+.container {
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	width: 100%;
+}
+svg {
+	width: 50rem;
+	height: auto;
+}
+#handboy {
+	animation: swing ease-in-out 1.3s infinite alternate;
+	transform-origin: 98% 98%;
+    @include transbox(fill-box);
+}
+#girllight {
+	animation: swing ease-in-out 1.3s infinite alternate;
+	transform-origin: 0% 97%;
+	@include transbox(fill-box);
+}
+#hairgirl {
+	animation: swinghair ease-in-out 1.3s infinite alternate;
+	transform-origin: 60% 0%;
+	@include transbox(fill-box);
+}
+#zero {
+	transform-origin: bottom;
+	@include transbox(fill-box);
+}
+.backbtn {
+	display: flex;
+	justify-content: center;
+	align-items: center;
+}
+.btn {
+	font-family: 'Nunito', sans-serif;
+	line-height: 0;
+	padding: .8rem 1rem;
+	border-radius: 8em;
+	transition: 0.5s;
+	margin-top: 2rem;
+	color: #fff;
```

Comment:
We are using scss so declaring variables should be the way for colors and fonts!